README.md

```diff
@@ -101,23 +101,31 @@ git clone git@github.com:roboflow-ai/roboflow-python.git && cd roboflow-python
 # Be sure to update the values with your account's information
 cp .env-example .env
```

Comment:
```suggestion
```

we can remove this and the comment above it.

we can also remove the `.env-example` file and everything about the `.env` variables from the `Testing` section below.